web/html/src/branding/css/susemanager-light.less

```diff
@@ -1,32 +1,5 @@
 @import url(bootstrap/less/bootstrap.less);
 
-/* The basic default css and the overriding customization right after each */
-@import url(susemanager-typography.less);
-@import url(variables.less);
-@import url(susemanager-light-variables.less);
-@import url(theme.less);
-@import url(susemanager-theme.less);
-@import url(mixins.less);
-@import url(susemanager-mixins.less);
-@import url(inputs.less);
-@import url(tables.less);
-@import url(error.less);
-@import url(tools.less);
-@import url(setup-wizard.less);
-@import url(visualization.less);
-@import url(notifications.less);
-@import url(fixes.less);
-@import url(network.less);
-@import url(modal.less);
-@import url(buttons.less);
-@import url(susemanager/buttons.less);
-@import url(susemanager/alerts.less);
-@import url(susemanager/panels.less);
-@import url(susemanager/nav.less);
-@import url(susemanager/titles.less);
-@import url(susemanager/toolbar.less);
-@import url(susemanager/inputs.less);
-@import url(susemanager/tables.less);
-
-// LEAVE THIS LINE AT THE END ALWAYS TO WARRANTY THE RESPONSIVE RULES REWRITE THE EXISTING STYLES
-@import url(responsive-rules.less);
+@import url(susemanager/variables.less);
```

Comment:
Maybe this could be moved into `susemanager/susemanager-theme.less` as well, and drop it from here and from `susemanager-dark.less`?

Btw, well done with this refactoring and centralizing what is common and shared. I remember when I started a while ago with this long journey, there is still a lot to do, but it already looks much much better now with this changes you are making they are very inline with the target, TYVM :heart_decoration:

Reply:
I originally wanted to do that, but I left it as is because this way we can have shared variables first (`susemanager/variables.less`), the any overrides if we need them for specifically for light/dark theme (`susemanager/susemanager-light-variables.less`) and then the theme. Does this sound ok for you?

Reply:
A-ha! I was thinking exactly like you said, but I didn't realized that `susemanager/susemanager-theme.less` is imported *after* `susemanager/susemanager-light-variables.less` so my suggestion wouldn't work because `susemanager/variables.less` would overrides the specific light/dark one. It is tricky and sad we cannot simplify it further, but that's how it is. Forget about my comment then. :+1: